### PR TITLE
Add Dynamixel driver library

### PR DESCRIPTION
**Pull request by hierophect**
This library adds support for the Dynamixel AX-12 servo motors. It also includes documentation on creating a simple direction control circuit for the required half-duplex UART.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -79,6 +79,9 @@
 [submodule "libraries/helpers/gamblor21_ahrs"]
 	path = libraries/helpers/gamblor21_ahrs
 	url = https://github.com/gamblor21/Gamblor21_CircuitPython_AHRS.git
+[submodule "libraries/drivers/dynamixel"]
+	path = libraries/drivers/dynamixel
+	url = https://github.com/hierophect/Circuitpython_Dynamixel.git
```
Comment (ladyada) [inline .gitmodules:84]: Can you make the URL somethin like Hierophect_CircuitPython_Dynamixel?

**Review by ladyada — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (hierophect) [reply .gitmodules:84]: Sure. I was trying to figure out the best convention for it from the other libraries in the bundle - I'm not making this on behalf of a personal LLC or anything yet, and it seemed like most other people just left out the prefix if they were doing it independently. I could leave a note on the Guide page that it's recommended to always use a prefix too, if you like.

Comment (ladyada) [reply .gitmodules:84]: probably for the best, so the authorship is in the title - i can also merge without

**Comment by hierophect** [comment]
@ladyada anything else you'd like me to do?

**Review by ladyada — APPROVED** [review]

**Comment by ladyada** [comment]
all good!

**Pull request merged**